account_manager/src/wallet/create.rs

```diff
@@ -126,30 +139,84 @@ pub fn create_wallet_from_mnemonic(
         .map_err(|e| format!("Unable to open --{}: {:?}", BASE_DIR_FLAG, e))?;
 
     // Create a random password if the file does not exist.
```

Comment:
We might want to move the comment down a few lines, it's not completely accurate here.